### ISSUE TITLE
Add Ability to View Restaurants When Offline

### ISSUE THREAD

**Issue by jfc3** [jfc3/10kapart#101]
Need to look into being able to view restaurants when offline.

Need to talk with Jason Garber and look at how Jeremy Keith made his entire book readable if offline since it had multiple chapters which are similar to me having multiple restaurants.

**Comment by jfc3** [comment]
Need to look into service workers and read over Jeremy Keith's JavaScript - https://resilientwebdesign.com/js/scripts.js to help find out more information on how to get all the places to eat or at least the city you're wanting in case the internet goes out or you have a poor connection.

**Comment by jfc3** [comment]
Being that this repository is the same as the [ATE HERE](https://github.com/jfc3/atehere) one only with smaller JSON file sizes to keep the total page load below 10K, I have stopped doing double the work.

If you want to see, the application working live go to [John F Croston III Ate Here](http://jfciii.com/atehere/).

There will be no further code updating at this time unless I try something crazy and don't want to mess with the live version.

Closing out all the open issues too.
